### ISSUE TITLE
SSR Build fail in swiper 6.8.2

### ISSUE THREAD

**Issue by rickvandermey** [nolimits4web/swiper#4833]
### Check that this is really a bug

- [X] I confirm

### Reproduction link

https://github.com/rickvandermey/angular-starterkit/tree/feature/Swiper_SSR 

After checkout and `npm ci` run `npm run ssr:build` to get the error

### Bug description

After upgrading from 6.7.5 to 6.8.2 the ssr build fails. 
There are not any code changes.

normal import the module, and declare the swiperCore according docs

html:
```
<swiper
	class="carousel"
	[slidesPerView]="1"
	[navigation]="true"
	[attr.test-id]="'deal-carousel'"
>
	<ng-container
		*ngFor="let photo of dealSync.photos; index as i"
	>
		<ng-template swiperSlide>
			<div class="gradient">
				<img
					class="image"
					[src]="photo.file + '&w=2560&h=861'"
					[attr.srcset]="
						buildImageSrcSet(photo.file)
					"
					[attr.alt]="
						photo.languageSettings[language]
							?.altText
					"
					[attr.test-id]="'carousel__image-' + i"
				/>
			</div>
		</ng-template>
	</ng-container>
</swiper>
```

```
An error occurred during the build:
Error: The target entry-point "swiper_angular" has missing dependencies:
 - swiper/types

    at TargetedEntryPointFinder.findEntryPoints (/Users/Sources/platform-nx/node_modules/@angular/compiler-cli/ngcc/src/entry_point_finder/targeted_entry_point_finder.js:40:23)
    at /Users/Sources/platform-nx/node_modules/@angular/compiler-cli/ngcc/src/execution/analyze_entry_points.js:29:41
    at SingleProcessExecutorSync.SingleProcessorExecutorBase.doExecute (/Users/Sources/platform-nx/node_modules/@angular/compiler-cli/ngcc/src/execution/single_process_executor.js:28:29)
    at /Users/Sources/platform-nx/node_modules/@angular/compiler-cli/ngcc/src/execution/single_process_executor.js:57:59
    at SyncLocker.lock (/Users/Sources/platform-nx/node_modules/@angular/compiler-cli/ngcc/src/locking/sync_locker.js:34:24)
    at SingleProcessExecutorSync.execute (/Users/Sources/platform-nx/node_modules/@angular/compiler-cli/ngcc/src/execution/single_process_executor.js:57:27)
    at Object.mainNgcc (/Users/Sources/platform-nx/node_modules/@angular/compiler-cli/ngcc/src/main.js:74:25)
    at Object.process (/Users/Sources/platform-nx/node_modules/@angular/compiler-cli/ngcc/index.js:29:23)
    at NgccProcessor.processModule (/Users/Sources/platform-nx/node_modules/@ngtools/webpack/src/ngcc_processor.js:175:16)
    at /Users/Sources/platform-nx/node_modules/@ngtools/webpack/src/ivy/host.js:146:18
    at /Users/Sources/platform-nx/node_modules/@ngtools/webpack/src/ivy/host.js:76:24
    at Array.map (<anonymous>)
    at Object.host.resolveModuleNames (/Users/Sources/platform-nx/node_modules/@ngtools/webpack/src/ivy/host.js:74:32)
    at actualResolveModuleNamesWorker (/Users/Sources/platform-nx/node_modules/typescript/lib/typescript.js:109497:133)
    at resolveModuleNamesWorker (/Users/Sources/platform-nx/node_modules/typescript/lib/typescript.js:109760:26)
    at resolveModuleNamesReusingOldState (/Users/Sources/platform-nx/node_modules/typescript/lib/typescript.js:109857:24)
An unhandled exception occurred: The target entry-point "swiper_angular" has missing dependencies:
 - swiper/types
 ```

### Swiper version

6.8.2

### Platform/Target and Browser Versions

terminal

### Validations

- [X] Follow our [Code of Conduct](https://github.com/nolimits4web/swiper/blob/master/CODE_OF_CONDUCT.md)
- [X] Read the [docs](https://swiperjs.com/swiper-api).
- [X] Check that there isn't already an issue that request the same feature to avoid creating a duplicate.
- [X] Make sure this is a Swiper issue and not a framework-specific issue

### Would you like to open a PR for this bug?

- [ ] I'm willing to open a PR

**Comment by jrstnly** [comment]
I am also seeing this issue. After downgrading to 6.8.1 it goes away.

**Comment by henkkelder** [comment]
same here

**Comment by jwjuele** [comment]
same here ;)

**Comment by henkkelder** [comment]
I can understand it that when a single user reports a weird problem that an example must be provided. 
In this case multiple users are reporting the same problem. In dutch we say: "Where there is smoke, there must be fire...".

**Comment by rickvandermey** [comment]
> Hello @rickvandermey. Please provide a online reproduction by [codesandbox](https://codesandbox.io/) or a minimal GitHub repository. You can fork one of our [demos](https://swiperjs.com/demos) in codesandbox to get start. Issues labeled by `missing demo` will be closed if no activities in 3 days.

Hi @nolimits4web , i've created a feature branch in my starterkit which will show the error

**Comment by RenAlbiorix** [comment]
I am also facing same error on serve too.

**Comment by itseddieschz** [comment]
In old versions and new versions that happens, I tried to install the old version that worked for me, but is also broken. I see directories in file contents and a folder called __ivy_ngcc__ is missing...

**Comment by itseddieschz** [comment]
Try with **npm i swiper@6.8.1** This works for now! 
But the new version still doesn't work properly.

**Comment by RenAlbiorix** [comment]
Yes. Version 6.8.1 works.

**Comment by nolimits4web** [comment]
should be fixed in 6.8.3

**Comment by hieuthanhpy** [comment]
> I am also seeing this issue. After downgrading to 6.8.1 it goes away.

Thanks, It's working
